core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1871,6 +1868,16 @@
    */
   public static final String CACHE_LEVEL_DEFAULT_VALUE = "BLOCK";
 
+  /**
+   * It is internal configuration and used only for test purpose.
```

Comment:
Internal ?  then for users, how to enable and disable this feature ?

Reply:
We are not giving this to user. For user, this will be always TRUE. User should not change it.